__DEFINES/setup.dm

```diff
@@ -577,6 +577,7 @@ var/list/global_mutations = list() // list of hidden mutation things
 #define CANKNOCKDOWN	2
 #define CANPARALYSE	4
 #define CANPUSH		8
+#define PACIFIABLE 16		//Can not be pacified
```

Comment:
This comment is neither helpful nor correct

Reply:
dang, forgot to update it